lib/config/presets/internal/monorepo.ts

```diff
@@ -223,6 +223,7 @@ const repoGroups = {
   webdriverio: 'https://github.com/webdriverio/webdriverio',
   workbox: 'https://github.com/googlechrome/workbox',
   xterm: 'https://github.com/xtermjs/xterm.js',
+  'xunit-dotnet': 'https://togithub.com/xunit/xunit',
```

Comment:
Do you _really_ mean to use the `togithub.com` domain? The others usually use `github.com`.

Reply:
```suggestion
  'xunit-dotnet': 'https://github.com/xunit/xunit',
```

🙈

Reply:
copy'n'paste from renovate PR